cmd/sharded-test-server/shard.go

```diff
@@ -62,8 +62,8 @@ func startShard(ctx context.Context, n int, args []string, servingCA *crypto.CA,
 	if n > 0 {
 		args = append(args, fmt.Sprintf("--shard-name=shard-%d", n))
 		args = append(args, fmt.Sprintf("--root-shard-kubeconfig-file=%s", filepath.Join(workDirPath, ".kcp-0/admin.kubeconfig")))
-		args = append(args, fmt.Sprintf("--embedded-etcd-client-port=%d", 2379+n+1))
-		args = append(args, fmt.Sprintf("--embedded-etcd-peer-port=%d", (2379+n+1)+1)) // prev value +1
+		args = append(args, fmt.Sprintf("--embedded-etcd-client-port=%d", 2380+(n*(n+1)/2)))
```

Comment:
Would you be willing to make this a helper function with a unit test?

Reply:
done, ptal

Reply:
Thank you! ❤️